src/components/core/Header.jsx

```diff
@@ -1,30 +1,41 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './Header.scss';
 import githubIcon from '../../images/github-icon.svg';
 import githubIconWhite from '../../images/github-icon-white.svg';
-import Toggle from 'react-toggle';
+import DarkModeToggle from 'react-dark-mode-toggle';
 
-const Header = ({ onToggleChangeDarkMode, darkMode }) => (
-  <div className="page-header">
-    <h2 className="page-header-title">
-      <span className="page-header-title-highlight">Issue</span>-Collab
-    </h2>
-    <div>
-      <Toggle
-          onChange={onToggleChangeDarkMode}
-      /> 
-      <span className="toggle-label">dark mode</span>
+const Header = ({ onToggleChangeDarkMode, darkMode }) => {
+  const [isDarkMode, setIsDarkMode] = useState(() => false);
```

Comment:
Hmm, we are storing darkMode state in App and it's being passed to Header. Do we need another instance of isDarkMode here? Can we just use the darkMode prop?

Reply:
It really works! I ended up letting it go unnoticed.